src/commands/utils/azurestorage.ts

```diff
@@ -40,23 +39,23 @@ export function getSASKey(
     startDate.setTime(startDate.getTime() - sasDuration(LinkDuration.StartTime));
     expiryDate.setTime(expiryDate.getTime() + sasDuration(linkDuration));
 
-    const AccountSasConstants = storage.Constants.AccountSasConstants;
-    const permissionsForSas = sasPermission(linkDuration, AccountSasConstants.Permissions);
-
-    const sharedAccessPolicy = {
-        AccessPolicy: {
-            Services: AccountSasConstants.Services.BLOB,
-            ResourceTypes: AccountSasConstants.Resources.SERVICE +
-                AccountSasConstants.Resources.CONTAINER +
-                AccountSasConstants.Resources.OBJECT,
-            Permissions: permissionsForSas,
-            Start: startDate,
-            Expiry: expiryDate
-        }
-    };
+    const permissionsForSas = sasPermission(linkDuration);
+
+    // Please refer to the new implementation of azure-storage api here: https://github.com/Azure/azure-storage-node 
```

Comment:
Readers of the code aren't very interested in how storage used to work.  Focus on what is current e.g. "The Azure storage client doesn't export constants.  The ones we declare here are copied from <permalink>."  (Also use a GitHub permalink rather than a line in the master branch, because code in master will move around over time.)